src/celldown.coffee

```diff
@@ -242,7 +254,14 @@ celldown = do () ->
                         cell = firstChar + ("-" for [1..size-2]).join("") + lastChar
                     else
                         missingChars = size - cell.length
-                        if missingChars > 0 then cell += (" " for [1..missingChars]).join("")
+                        # Used only if alignment isn't center
+                        spaces = if missingChars > 0 then (" " for [1..missingChars]).join("") else ""
+                        console.log @arr[rowIndex][colIndex]
```

Comment:
Could you please remove this console.log?

Reply:
I'm sorry, I forgot to remove it.